Dockerfile.tools

```diff
@@ -0,0 +1,25 @@
+FROM registry.access.redhat.com/ubi8/ubi:8.1 AS build
+
+WORKDIR /
+
+# Install deps
+RUN dnf update -y && dnf install -y make curl golang git
+
+COPY hack/make/dependencies.make hack/make/dependencies.make
+COPY Makefile Makefile
+
+# run make
+RUN make kustomize operator-sdk helm yq clusteradm LOCALBIN=/usr/local/bin
+
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.8 AS tools
+
+RUN microdnf install -y git && microdnf clean all
+
+COPY --from=build /usr/local/bin /usr/local/bin
+COPY --from=build /usr/bin/clusteradm /usr/bin/clusteradm
+COPY --from=build /usr/bin/clusteradm /usr/bin/clusteradm
```

Comment:
Is this meant to be here twice? Why is it not in `/usr/local/bin`?

Reply:
Whoops! Will remove the duplicate now. Not sure why `make` installed `clusteradm` to this directory. Do you think I should copy it into the `/usr/local/bin` directory?

Reply:
Seems the `make` target for `clusteradm` doesn't use the `LOCALBIN` variable for some reason.
```
.PHONY: clusteradm
clusteradm: $(CLUSTERADM)
$(CLUSTERADM):
	test -s $(CLUSTERADM)|| curl -sL https://raw.githubusercontent.com/open-cluster-management-io/clusteradm/main/install.sh | INSTALL_DIR=bin bash -s -- $(CLUSTERADM_VERSION)
```

Reply:
OK, odd, not something we need to fix here i guess. Remove the dup line and i think we can just merge it and see how it goes.